.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,13 @@
-<!--
-See http://loopback.io/doc/en/contrib/triaging-pull-requests.html to understand
-the MINIMUM requirements for submitting pull requests.
--->
+### Description
 
-### Type of change
-
-<!--
-See http://semver.org/ for more info.
--->
-
-- [ ] SEMVER-MAJOR -- Backwards-incompatible changes
-- [ ] SEMVER-MINOR -- Add functionality in a backwards-compatible manner
-- [ ] SEMVER-PATCH -- Backwards-compatible bug fixes
-
-### Description of change(s)
-
-
-
-### Link(s) to related GitHub issues
 
+#### Related issues (ie. strongloop/loopback#49, etc)
 
+- None
 
 ### Checklist
 
 - [ ] CLA signed
-- [ ] Commit message conforms with the [Git commit message
-  guidelines](http://loopback.io/doc/en/contrib/git-commit-messages.html)
-- [ ] New tests are added to cover all changes
-- [ ] Code passes `npm test`
```

Comment:
@bajtos @raymondfeng @ritch I'm proposing we remove the commit message, new tests added and code passes npm test items as "All CI builds are passing" already checks these 3 items for us. If CI fails, one of those 3 items failed anyways. Thoughts? ie. reduce to:

```
- [ ] CLA signed
- [ ] Code conforms with the [style
   guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] All CI Builds are passing
```

The style guide requirements are subjective but might be worth leaving in since we'll be asking them to check each time we review anyways.